### PR TITLE
Customize Navbar

### PR DESCRIPTION
**Pull request by IcedTea2K**
- Swizzle navbar component to more customization

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -89,25 +89,29 @@ const config = {
...
+          //   label: 'Welcome',
+          // },
+          // {
+          //   to: 'resources/intro',
+          //   docId: 'intro',
+          //   label: 'Resources',
+          //   position: 'right',
+          // },
+          // {to: '/blog', label: 'Blog', position: 'right' },
+          {to: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ', label: 'Modules', position:'right'},
```
Comment (Kevin-Tan25) [inline docusaurus.config.js:111]: i knew this was coming but i still clicked onto it :o

**Review by Kevin-Tan25 — COMMENTED** [review]
could you also remove light theme? I think website is sufficient with dark theme only for now

File: docusaurus.config.js [hunk 2]
```diff
@@ -121,7 +125,7 @@ const config = {
       // },
       colorMode: {
         defaultMode: 'dark',
```
Comment (IcedTea2K) [inline docusaurus.config.js:127]:
```suggestion
        defaultMode: 'dark',
        disableSwitch: true,
        respectPrefersColorScheme: false,
```
Only dark mode would be enabled with this.

**Review by Kevin-Tan25 — APPROVED** [review]
LGTM

**Pull request merged**